massa-execution-worker/src/execution.rs

```diff
@@ -1124,39 +1124,12 @@ impl ExecutionState {
         &self,
         addr: &Address,
     ) -> (BTreeSet<Vec<u8>>, BTreeSet<Vec<u8>>) {
-        // here, get the final keys from the final ledger, and make a copy of it for the candidate list
```

Comment:
I recommend putting this back here instead of a separate "common" file

Reply:
done